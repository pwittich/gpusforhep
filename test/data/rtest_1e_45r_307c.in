044bd5
08f2e2
08dcb5
0d85cc
100cb4
11e1f8
15feed
00c625
378400
05be61
0538ad
09919a
0d2565
0d0492
10b3fb
106812
15feed
00ee03
2aac3e
04f4c6
08b9e4
090d00
0c6b39
0cbf75
118532
1112e0
15feed
02cc24
36aa5c
04a5d7
09eadf
091960
0df9e9
11966d
1098a0
15feed
023b7c
15feed
02c448
2dfea1
053244
09518a
09d660
0d45a5
119ac2
1002ee
15feed
03e7e0
15feed
021802
3e4aea
046619
0974ab
0d396d
11c27a
15feed
01eaa7
371f3d
05236e
0582fd
0912eb
091ad3
0c636b
0cc9e0
100b3c
10e0c8
15feed
032818
387c2d
044516
0913fa
087163
0cd8e3
11994a
15feed
01dacd
3aabbb
056510
093fdf
087dea
0ca582
11cbcf
15feed
002930
15feed
03f5e2
2b25e6
05454a
05c90d
086a2b
09536f
0dc618
1199b4
15feed
008019
36f2ca
044a8c
088383
0c3549
0c569b
11e96a
106413
15feed
03e79a
15feed
00d7b6
229d79
047a65
08f67a
0dff78
0cbc2f
105c76
117d1a
15feed
012e4c
248365
04e66b
04b58d
09f4d8
096634
0cca84
101568
10445a
15feed
011e7a
15feed
03930f
36e96c
0539ac
048e4d
0883d6
0ce2b5
11ef50
15feed
0282f5
15feed
02f299
399264
04bd86
05bf97
08a12f
09510c
0d72b2
104274
107aca
15feed
016c42
2e7648
0438d4
0938a2
095ceb
0c3e34
0cc342
10833b
15feed
022fc7
343215
047b6d
05cfe6
0916bb
0d20e5
102d20
10488d
15feed
037ae8
2fbc5e
04bf7f
048711
09b3b5
09bb70
0d8ee2
0df7fe
1045bb
15feed
01ab1f
3d5285
04a594
0424c9
08d685
082439
0d4908
10648e
10ca4c
15feed
00538f
15feed
02f0d6
3f8d75
05e241
055c65
08db5a
0dad1c
0d7092
1149bf
15feed
036b94
37ca9e
047063
0411a8
09301c
0d04c4
116430
10a96f
15feed
014085
20b030
05b1c1
092ca8
084508
0d701a
0cc91c
103427
114f66
15feed
0009ac
24030e
04a9dd
095f92
096e6e
0d1a8a
10764e
15feed
0271e7
15feed
03f521
3a394c
0459a0
049491
0839cb
09bb84
0c7808
11de3b
1060ae
15feed
03651e
39ea2a
04a831
058923
08b4e2
0df932
0df4a9
102f7c
15feed
01fb92
15feed
031aa4
271e6a
041689
047586
09f450
093328
0d9ff3
1143f9
11dc1d
15feed
001c0f
2299d6
04e2c5
041750
09a01c
08e2fc
0ccb80
0cb016
11773e
15feed
01abcb
15feed
00f1bb
2e8c3b
04b1f6
0513ff
0909f4
0d7fff
0dac4e
10cb90
15feed
008d3c
15feed
0361ea
3cc0c5
048850
047d84
09bb45
0df82f
110623
11e8e2
15feed
023b48
15feed
03fea7
27909e
04e48b
08af0b
0c14d5
0d46dd
11af19
15feed
02c09e
3ee38c
05d1ce
08695e
080e41
0c53b0
115d6e
15feed
02f934
15feed
02d8fe
2fe0ad
04722f
08af44
089302
0cecb5
117c5e
15feed
01601e
300d21
044c6a
041483
09d980
0d868f
106062
15feed
024124
15feed
015ccb
3f09b1
054068
09219f
091606
0dfa71
105886
11f567
15feed
0282c6
387be2
0449f8
09df5c
0ca6ef
102744
15feed
02d9ae
15feed
02c78a
37b836
043600
04ba7b
09e15f
09e34e
0c1641
0c06e5
10f289
10209e
15feed
027462
15feed
03a15e
307a4e
04f423
046342
0824fc
0c6249
0d7e8f
115e98
15feed
00b43e
15feed
0358b8
36f005
054d0f
054bea
0917e2
08700c
0d10db
0cbebc
1019ba
15feed
023070
26b72a
05f948
08f9be
0c6323
1012d2
108086
15feed
0316d8
22c8c1
051dc2
04d0f4
084f62
0d83d8
11411a
15feed
00f76e
2bfd26
05c0df
08bc27
087091
0d908b
1134d4
15feed
02ebfe
27ec3a
0495f3
050629
08d287
0994e4
0cdf57
11a225
11e428
15feed
02466b
15feed
0267a8
31fe74
05e3df
08bf28
0c9c2b
0d9972
111b7c
15feed
01abb8
15feed
013dc8
269afc
050d72
094f4e
0d24ed
0cebd7
1009a1
15feed
00d709
34e19b
05ef94
091f24
0ca5be
110f5e
15feed
033437
15feed
020e08
33d8d9
7fe9b8
