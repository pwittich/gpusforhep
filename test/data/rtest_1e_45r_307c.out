7fe8b8
